### PR TITLE
Test Change

### PR DESCRIPTION
**Pull request by ohpyupi**
### Checklist
- [ ] Unit Testing

### CODE REVIEW

File: src/pages/dog-page/dog-page.test.js [hunk 1]
```diff
@@ -3,7 +3,6 @@ import DogPage from './dog-page.js';
 import Loading from '../../components/loading/loading.js';
 import { shallow, mount } from 'enzyme';
 import DogCard from '../../components/dog-card/dog-card.js';
-// import { waitFor } from '@testing-library/react';
```
Comment (jyon124) [inline src/pages/dog-page/dog-page.test.js:6]: Why did you delete this line?

Comment (ohpyupi) [reply src/pages/dog-page/dog-page.test.js:6]: Removed it since its not being used

**Review by jyon124 — APPROVED** [review]
👍

**Comment by jyon124** [comment]
Testing Purpose
